build.gradle

```diff
@@ -191,6 +192,14 @@ project(':iceberg-bundled-guava') {
 }
 
 project(':iceberg-api') {
+  apply plugin: 'com.palantir.revapi'
+
+  revapi {
+    oldGroup = 'org.apache.iceberg'
+    oldName = 'iceberg-api'
+    oldVersion = '0.13.0'
+  }
```

Comment:
Using this seems to be not the best idea, as it would need to be updated manually across versions.

Using the gradle tasks doesn't update this.